lhotse/supervision.py

```diff
@@ -512,6 +512,50 @@ def from_dicts(data: Iterable[Dict]) -> "SupervisionSet":
             SupervisionSegment.from_dict(s) for s in data
         )
 
+    @staticmethod
+    def from_rttm(path: Pathlike) -> "SupervisionSet":
```

Comment:
It would be nice to add a short description of RTTM format in this docstring (which column stands for what)